docs/source/guides/character_control_guide.rst

```diff
@@ -0,0 +1,159 @@
+
+.. _`Contribution Guide`: https://docs.nucypher.com/en/latest/guides/contribution_guide.html/
+.. _`Discord`: https://discord.gg/7rmXa3S
+
+If you are ever given a 500 status code, we'd like to know about it!
+You can see our `Contribution Guide`_ for getting involved.
+Ideally, you can share some information with us about what you were doing when you encountered the 500 in the form of a GitHub issue, or just tell us in our `Discord`_.
+
+HTTP Methods
+------------
+Currently, the character control API only uses the following HTTP methods:
+
+- ``POST``
+- ``PUT``
+
+We don't exactly follow RESTful methodology precisely.
+Take careful note following the API endpoints to see what to send and what to expect as a response.
+
+Request Format
+--------------
+The character control API uses JSON for all its endpoints. A request may look like:
+
+.. code::
+
+    {
+        'bob_encrypting_key': '0324df67664e6ea40f2eea8037c994debd4caa42117fe86cdb8cab6ac7728751ad',
+        'label': 'dGVzdA==',
+        'm': 2,
+        'n': 3,
+        'expiration_time': '2019-02-14T22:23:10.771093Z',
+    }
+
+Take a look at ``bob_encrypting_key``. Take note that it's a hex-encoded string.
+The character control API endpoints expect `all` keys to be encoded as hex.
+
+Now, look at ``label``. Notice that it's a base64-encoded string.
+Whenever the Python API expects the ``bytes`` type, the character control API will expect a base64 encoded string.
+
+Integers, in our case ``m`` and ``n`` can be passed as is without encoding.
+
+A datetime, like ``expiration_time``, must be passed in as an ISO-8601 formatted datetime string.
+
+If you are missing a required argument in your request, you will be returned a 400 status code.
+
+Response Format
+---------------
+Like we determined above, the character control API uses JSON for all its endpoints.
+The same goes for our API's responses. One may look like:
+
+.. code::
+
+    {
+        'result': {
+            'treasure_map': 'Y8Wl+o...Jr4='
+        }
+    }
+
+The character control API will return the results of our Python API.
+If any data is returned, like a treasure map or a message kit, it will be serialized as base64 with the object name being a key inside ``result``.
+
+Be sure to also check the returned status code of the request. All successful calls will be 200.
+See the above "Status Codes" section on what to do in the event of a 400 or 500.
+
+Character Control Endpoints
+===========================
+
+Alice
+-----
+
+grant
+~~~~~
+
+This endpoint controls the ``Alice.grant`` method.
+
+- URL: ``/grant``
+- HTTP Method: ``PUT``
+- Required arguments:
+    - ``bob_encrypting_key`` -- encoded as hex
+    - ``label`` -- encoded as base64
+    - ``m`` -- an integer
+    - ``n`` -- an integer
+    - ``expiration_time`` -- an ISO-8601 formatted datetime string.
+- Returns: a base64-encoded ``treasure_map``
+
+For more details on these arguments, see the nucypher documentation on the ``Alice.grant`` Python API method.
```

Comment:
Where is this? Is it not written yet?

Reply:
I'm.... not sure. I was just guessing we would add some at some point. It should probably direct folks to the code itself.